app/concerns/web_request_concern.rb

```diff
@@ -2,6 +2,18 @@
 require 'faraday_middleware'
 
 module WebRequestConcern
+  module DoNotEncoder
+    def self.encode(params)
+      params.map do |key, value|
+        value.present? ? "#{key}=#{value}" : "#{key}"
```

Comment:
Not sure, I just made sure @K1773R's [example](https://github.com/cantino/huginn/issues/854#issuecomment-118796264) works. In that case the `encode` function is passed `{GetRss?CategoryNr=39207' => nil}`. Rack parses your URL as `{"key"=>nil, "key3"=>nil}`